classes/task/FileLoader.inc.php

```diff
@@ -392,27 +392,6 @@ private function _stageFile()
     {
         $this->moveFile($this->_processingPath, $this->_stagePath, $this->_claimedFilename);
     }
-
-    /**
-     * Send the passed message to the administrator by email.
-     *
-     * @param $message string
-     */
-    private function _notify($message, $messageType)
```

Comment:
This method isn't being used, but there's another place using the old Mail class, which needs to be wiped off.